azavea/raster-foundry
# Add `planetCredential` field to user model

## Overview

This PR adds a `planetCredential` field the user model.

### Checklist

~- [ ] Styleguide updated, if necessary~


## Testing Instructions

 * Run migrations
 * Fetch your user's info from the `/users/{uuid}` endpoint and see that the credential field is present

Closes #2143
